.prettierignore

```diff
@@ -2,6 +2,11 @@ vendor
 
 # VSCode Extension
 extensions/vscode/out
+test/sample-content
 
 # webdriver.io service
 test/vscode-ui/.wdio-vscode-service
+extensions/vscode/dist
+extensions/vscode/node_modules
+extensions/vscode/webviews/homeView/node_modules
+extensions/vscode/webviews/homeView/dist
```

Comment:
Changes have been replicated into different PR, should clean up after some merges. https://github.com/posit-dev/publisher/pull/1948